src/System.ComponentModel.Annotations/tests/PhoneAttributeTests.cs

```diff
@@ -24,6 +24,21 @@ protected override IEnumerable<TestCase> ValidValues()
             yield return new TestCase(new PhoneAttribute(), "425-555-1212 ext.123");
             yield return new TestCase(new PhoneAttribute(), "425-555-1212 ext. 123");
             yield return new TestCase(new PhoneAttribute(), "1");
+
+            // Certain invalid phone numbers are reported as valid with .NET core.
+            // The full .NET framework considers them invalid. This is likely a bug
+            // in .NET core.
+            if (!PlatformDetection.IsFullFramework)
```

Comment:
I believe we break the test into two (possibly with a shared body) so they can have different member data.